extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaProcessor.java

```diff
@@ -108,12 +111,24 @@
     };
 
     static final DotName OBJECT_MAPPER = DotName.createSimple("com.fasterxml.jackson.databind.ObjectMapper");
+    private static final Set<String> SASL_PROVIDERS = Arrays.stream(new String[] {
+            "com.sun.security.sasl.Provider",
+            "org.apache.kafka.common.security.scram.internals.ScramSaslClientProvider",
+            "org.apache.kafka.common.security.oauthbearer.internals.OAuthBearerSaslClientProvider"
+    }).collect(Collectors.toSet());
```

Comment:
Just for future reference, now that we have moved to Java 11 we have goodies like `Set.of(...)` to replace this kind of thing